### ISSUE TITLE
Ansible Service Broker is broken with default image in 3.11

### ISSUE THREAD

**Issue by tenortim** [openshift/openshift-ansible#11213]
#### Description

This is pretty much identical to issue 9960 but for 3.11 instead of 3.10:
By default, 3.11 will deploy the latest asb image from upstream: https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/ansible_service_broker/defaults/main.yml#L27

It's not clear which release this is/was targeted for but it doesn't work correctly for 3.11. The default value in openshift-ansible should deploy a broker specifically built for 3.11 and not follow a floating latest. The latest 3.11 image is: `docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.3.22-1`.

##### Version

```
[timw@centos7-timw openshift-ansible]$ ansible --version
ansible 2.6.5
  config file = /home/timw/src/openshift-ansible/ansible.cfg
  configured module search path = [u'/home/timw/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Oct 30 2018, 23:45:53) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
[timw@centos7-timw openshift-ansible]$ git describe
v3.11.0
```

##### Steps To Reproduce
Deploy a cluster with ansible-service-broker enabled, and accept the default origin broker image.

##### Expected Results
The service catalog should be able to contact the broker successfully.

##### Observed Results
The service catalog 404s when requesting the broker's catalog. The following command will display the 404 error under status:

`oc describe clusterservicebroker $ASB_BROKER_NAME`

**Comment by tenortim** [comment]
The following change eliminated the problem in my environment:
```
--- a/roles/ansible_service_broker/defaults/main.yml
+++ b/roles/ansible_service_broker/defaults/main.yml
@@ -24,7 +24,7 @@ ansible_service_broker_local_registry_whitelist: []
 ansible_service_broker_local_registry_namespaces: ["openshift"]

 l_asb_default_images_dict:
-  origin: 'docker.io/ansibleplaybookbundle/origin-ansible-service-broker:latest'
+  origin: 'docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.3.22-1'
   openshift-enterprise: 'registry.redhat.io/openshift3/ose-ansible-service-broker:${version}'

 l_asb_default_images_default: "{{ l_asb_default_images_dict[openshift_deployment_type] }}"
```

**Comment by DennisKonrad** [comment]
Hi,
we are experiencing the same problem with 3.11. We are trying to use the fix @tenortim mentioned.

Is there are reason openshift-enterprise uses a fixed version?

**Comment by dbarreda** [comment]
I can confirm that @tenortim solution works.

**Comment by BostjanBozic** [comment]
I was experiencing same issue with OKD 3.11 - it seems issue was only in tag itself. It as set to `v3.11.0`; once changed to `v3.11` it deployed without issues.

**Comment by matsstaff** [comment]
Ran into this issue. Could not find any info on _how_ to actually fix it without redeploying the cluster. Turns out all I needed to do was edit the deployment config of asb and change the image to docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.3.22-1 (and redeploy asb if not done automatically on config change).
I thought I needed to redeploy through some ansible playbook script, and spent more time than I'd like to admit researching that.
